docs/deploy-to-cluster/signup-aws.rst

```diff
@@ -0,0 +1,56 @@
+.. _signup-aws-to-cluster:
+
+=================================================
+Sign up for CrateDB Cloud via the AWS Marketplace
```

Comment:
The title is somewhat misleading. The user doesn't "sign up" for CC via the AWS MP. They "deploy a cluster" for CC via the AWS MP.

Reply:
It was the same in the other (original) tutorial. I think you're right though that it could be confusing with the Cloud signup process, so I'm changing it to 'subscribe to...'.